### DISCUSSION TITLE
Theme options for VisiData

### DISCUSSION THREAD

**Discussion by saulpw** [saulpw/visidata#1500] [Show and tell]
VisiData has so many configuration options, it can be hard to keep track of them all.  I'm starting this thread to show off the possibilities.  If you have custom display options set for VisiData, please post them here!

**Comment by saulpw** [comment]
I have this little snippet to change the icon for a VisiData type:

    vd.typemap[int].icon = 'ⓓ'

**Comment by saulpw** [comment]
Here's a light theme I threw together (from VisiData Plus): https://github.com/visidata/vdplus/blob/develop/lightscheme.py

**Comment by frosencrantz** [comment]
Here's one I mentioned before (https://github.com/saulpw/visidata/issues/887#issuecomment-774540356) that can help read multiline cells:

There are options for `disp_row_top`, `disp_row_mid`, `disp_row_bottom`, where you can use the unicode box drawing characters to make a bracket. There are different types of box drawing characters. For example:
╔
║
╚

**Comment by geekscrapy** [comment]
It'd be great if there were screenshots with these suggestions! Could do casual vd theme browsing 😜

**Reply by saulpw** [reply]
Great idea!  I'd love it if someone put together some screenshots.

**Comment by saulpw** [comment]
On `develop`,  just made a lightweight theme chooser, and added the `light` theme from vdplus, and made two new themes, `asciimono` and `ascii8`.  Check em out:

- ![asciimono](https://www.visidata.org/images/vd-theme-asciimono.png)
- ![ascii8](https://www.visidata.org/images/vd-theme-ascii8.png)
- ![light](https://www.visidata.org/images/vd-theme-light.png)
- ![default](https://www.visidata.org/images/vd-theme-default.png)

**Comment by AllergicMushroom** [comment]
Is there a way to create custom themes ? I don't see any in the documentation.

As a side note, my light theme is different from yours. The background is correct but everything that's supposed to be colored aside from the top bar is black. Am I missing something or is this a bug ?
